cfct/manifest.yaml

```diff
@@ -0,0 +1,121 @@
+      accounts:
+        - Audit
+    export_outputs:
+      - name: /org/sharedservice/networking/vpcflowlogsbucket
+        value: $[output_oBucketName]
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogsMgmtRole
+    resource_file: templates/vpc_flowlog_mgmt.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "EventBusDestinationAccount" # Typiclly this will be the NetworkHub account
+        parameter_value: "<REPLACE ME>"
+
+      - parameter_key: "BaselineCloudTrailStackArn" # Copy this this from mgmt account in cloudformation stacksets for AWSControlTowerBP-BASELINE-CLOUDTRAIL
+        parameter_value: "<REPLACE ME>"
+    deploy_method: stack_set
+    deployment_targets:
+      accounts:
+        - Management
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogRoleInSpoke
+    resource_file: templates/vpc_flowlog_spoke.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "EventBusDestinationAccount" # Network hub account
+        parameter_value: "<REPLACE ME>"
+
+      - parameter_key: "EventBusName"
+        parameter_value: "FlowLog-EventBus"
+
+      - parameter_key: "ControlTowerMasterRegion"
+        parameter_value: "ap-southeast-2"
+    deploy_method: stack_set
+    deployment_targets:
+      organizational_units:
+        - Security
+        - Infrastructure
+        - NonProduction
+        - Production
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogAutomationInHub
+    resource_file: templates/vpc_flowlog_automation_in_hub.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "MgmtAccountId"
+        parameter_value: "$[alfred_ssm_/org/core/ManagementAccountId]"
+
+      - parameter_key: "ControlTowerMasterRegion"
+        parameter_value: "ap-southeast-2"
+
+      - parameter_key: "FlowLogBucketName" 
+        parameter_value: "$[alfred_ssm_/org/sharedservice/networking/vpcflowlogsbucket]"
+
+      - parameter_key: "EventBusDestinationAccount" # Network hub account
+        parameter_value: "220969071716"
+
+      - parameter_key: "EventBusName"
+        parameter_value: "FlowLog-EventBus"
+
+      - parameter_key: "ComplianceFrequency" # how often to check for tag updates by schedule
+        parameter_value: "24"
+
+      - parameter_key: "BaselineCloudTrailStackSetName"
+        parameter_value: "AWSControlTowerBP-BASELINE-CLOUDTRAIL"  # do not change this, used to get all accounts managed by Control Tower.
+
+      - parameter_key: "BaselineCloudTrailStackArn"
+        parameter_value: "<REPLACE ME>"  # find this from mgmt account in cloudformation stacksets
+
+      - parameter_key: "S3LambdaBucket"
+        parameter_value: "$[alfred_ssm_/org/primary/storagebucket]"
```

Comment:
this SSM param does not exist by default, I suggest to make call out in the README.md

Reply:
removed use of alfred, for ease of deployment

Reply:
Now available via management_prep.template, which I call out in the readme as a pre-req